components/brave_vpn/brave_vpn_os_connection_api.h

```diff
@@ -125,8 +125,7 @@ class BraveVPNOSConnectionAPI
   bool needs_connect_ = false;
   bool prevent_creation_ = false;
   std::string target_vpn_entry_name_;
-  mojom::ConnectionState connection_state_ =
-      mojom::ConnectionState::DISCONNECTED;
+  mojom::ConnectionState connection_state_ = mojom::ConnectionState::UNKNOWN;
```

Comment:
looks like you have a rebase issue here, `UNKNOWN` was removed

Reply:
it was added in this PR, see comment above

Reply:
I think this change should be removed. We can assume disconnected until we detect that it is connected, but also how can this change be made and not break tests? We are definitely missing some very important test coverage for brave_vpn_os_connection_api is this change doesn't break anything

Reply:
removed in https://github.com/brave/brave-core/pull/15989/commits/80862768735dfc5ce4546e5d0608b31d51b044ef